benchmark/Snakefile

```diff
@@ -0,0 +1,537 @@
+
+seed_array = np.random.random_integers(1,2**31,replicates)
+
+def HuberDFE():
+    """
+    Return neutral and negative MutationType()s representing a human DFE.
+    Huber et al. (2017)
+    """
+    neutral = stdpopsim.ext.MutationType()
+    gamma_shape = 0.19  # shape
+    gamma_mean = -0.014  # expected value
+    h = 0.5  # dominance coefficient
+    negative = stdpopsim.ext.MutationType(
+        dominance_coeff=h,
+        distribution_type="g",  # gamma distribution
+        distribution_args=[gamma_mean, gamma_shape],
+    )
+    # neutral mutations have 0 proportion because they are not simulated by SLiM
+    return {"mutation_types": [neutral, negative], "proportions": [0.0, 0.7]}
+
+## Run only the simulations!
+#rule all:
+#    input: expand(output_dir + "/simulated_data/{seeds}/sim_OutOfAfrica_3G09_HuberDFE_{chrms}.trees", seeds=seed_array, chrms=chrm_list) 
+
+rule all:
+    input: 
+        #expand(output_dir + "/dadi/{seeds}/{chrms}/pop{ids}/pop{ids}.neu.fs", seeds=seed_array, chrms=chrm_list, ids=[0, 1, 2]),
+        #expand(output_dir + "/dadi/{seeds}/{chrms}/pop{ids}/pop{ids}.nonneu.fs", seeds=seed_array, chrms=chrm_list, ids=[0, 1, 2]),
+        #expand(output_dir + "/polyDFE/{seeds}/{chrms}/pop{ids}/pop{ids}.polyDFE.results", seeds=seed_array, chrms=chrm_list, ids=[0, 1, 2]),
+        #expand(output_dir + "/dfe-alpha/{seeds}/{chrms}/pop{ids}/pop{ids}.dfe-alpha.neu.config", seeds=seed_array, chrms=chrm_list, ids=[0, 1, 2]),
+        #expand(output_dir + "/dfe-alpha/{seeds}/{chrms}/pop{ids}/pop{ids}.dfe-alpha.nonneu.config", seeds=seed_array, chrms=chrm_list, ids=[0, 1, 2]),
+        #expand(output_dir + "/dfe-alpha/{seeds}/{chrms}/pop{ids}/pop{ids}.dfe-alpha.fs", seeds=seed_array, chrms=chrm_list, ids=[0, 1, 2]),
+        #expand(output_dir + "/dfe-alpha/{seeds}/{chrms}/pop{ids}/nonneu/est_dfe.out", seeds=seed_array, chrms=chrm_list, ids=[0, 1, 2]),
+        #expand(output_dir + "/dadi/{seeds}/{chrms}/pop{ids}/pop{ids}.two_epoch.InferDM.opts.0", seeds=seed_array, chrms=chrm_list, ids=[0, 1, 2])
+        #expand(output_dir + "/dadi/{seeds}/{chrms}/pop{ids}/pop{ids}.two_epoch.gamma.InferDFE.bestfits", seeds=seed_array, chrms=chrm_list, ids=[0, 1, 2])
+        #expand(output_dir + "/polyDFE/{seeds}/{chrms}/pop{ids}/pop{ids}.polyDFE.bestfit", seeds=seed_array, chrms=chrm_list, ids=[0, 1, 2])
+        #expand(output_dir + "/dadi/pop{ids}.dadi.dfe.bestfits", ids=[0, 1, 2]),
+        #expand(output_dir + "/polyDFE/pop{ids}.polyDFE.bestfits", ids=[0, 1, 2])
+        expand("plots/{chrms}/dfe.inference.benchmark.pdf", chrms=chrm_list)
+
+#rule all:
+#    input:
+#        "results/dadi/pop0.two_epoch_1d.gamma.mut.prop.pdf",
+#        "results/polyDFE/pop0.polyDFE.results",
+#        "results/dfe-alpha/nonneu/est_dfe.out",
+#        "results/anavar/pop0.anavar.ctl.txt",
+#        "results/anavar/pop0.anavar.results",
+#        "results/anavar/pop0.anavar.log",
+
+#rule all:
+#    input: "results/prfreq/pop0.prfreq.fs"
```

Comment:
what's all this?

Reply:
@petrelharp  These can be removed, I just used them to test different software separately.